sec/intro.tex

```diff
@@ -7,56 +7,59 @@ \section{Introduction}\label{sec:intro}
 research data directly. That article is one of many in an increasing body of
 publications over the last decade that concern themselves with the state of
 research data~\cite{Higman2019}, its utilisation and
-openness~\cite{Aslam2017,Zuiderwijk2020}, and the importance of good
+openness~\cite{Aslam2017,Zuiderwijk2020}, and the importance of best
 practices~\cite{Colavizza2020,Corti2019}. These publications, for the most part,
 do not cater specifically to the increasingly popular concepts attached to
-\emph{Big Data}, where the volume and variety of a data source should be
+\emph{Big Data}, where the volume and variety of a data source should be  % TODO I dont fully understand what we're saying here.
```

Comment:
@drvinceknight this is one of those points where I'm setting the seed for what comes in a later chapter (the EDO chapter) but I don't think it really adds anything. I'll delete it.

Reply:
This has been reworded in 5cfbf21 to:

```tex
practices~\cite{Colavizza2020,Corti2019}. These publications focus on using this
set of characteristics as a measure of the quality of a research data source,
which also acts as a mark of the associated research's quality; this is in
contrast to other recent trends where the pursuit of more voluminous and varied
data sources has taken precedence~\cite{Batistic2019}.
```

Reply:
How do you feel about this wording @drvinceknight?

Reply:
Yeah this is good :+1: